superset/config.py

```diff
@@ -263,6 +266,20 @@ class CeleryConfig(object):
 
     from superset_config import *  # noqa
     print('Loaded your LOCAL configuration')
+
+# email backend implementation
+email_backend = email.send_email_smtp
+# smtp server configuration
+smtp_host = 'localhost'
+smtp_starttls = True
+smtp_ssl = False
+smtp_user = 'superset'
+smtp_port = 25
+smtp_password = 'superset'
+smtp_mail_from = 'superset@superset.com'
+
+try:
+    from superset_config import *  # noqa
```

Comment:
no need to duplicate this, just move the smtp settings before the same statement above